src/core/Akka.Cluster/Routing/ClusterRoutingConfig.cs

```diff
@@ -298,7 +298,9 @@ public override int GetNrOfInstances(ActorSystem system)
         /// <inheritdoc cref="RouterConfig.CreateRouterActor"/>
         public override ActorBase CreateRouterActor()
         {
+#pragma warning disable AK1000
```

Comment:
Disabling `AK1000` here because custom router types kind of fall outside the normal use cases for not `new`-ing up actors